nuxeo-distribution/nuxeo-launcher/src/test/resources/log4j.xml

```diff
@@ -6,7 +6,7 @@
   <appender name="CONSOLE" class="org.apache.log4j.ConsoleAppender">
     <errorHandler class="org.apache.log4j.helpers.OnlyOnceErrorHandler" />
     <param name="Target" value="System.out" />
-    <param name="Threshold" value="WARN" />
+    <!-- <param name="Threshold" value="WARN" /> -->
```

Comment:
Please don't commit changes to the log4j configuration. Log output should be empty when running tests.